### PR TITLE
Add more descriptions for accounts package

### PR DESCRIPTION
**Pull request by andybclee**
## Proposed changes

- This PR adds more description for `accounts` package

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: accounts/doc.go [hunk 1]
```diff
@@ -18,5 +18,19 @@
 // This file is derived from accounts/accounts.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package accounts implements high level Klaytn account management.
+/*
+Package accounts implements high level Klaytn account management.
+
+There are several important data structures and their hierarchy is as below
+ Manager -- Backend -- Wallet -- Account
```
Comment (KimKyungup) [inline accounts/doc.go]: How about this? :)
```suggestion
 Manager -> Backend -> Wallet -> Account
```

File: accounts/doc.go [hunk 2]
```diff
@@ -18,5 +18,19 @@
...
+
+Source Files
+
+Each file provides the following features
+ - accounts.go	: Provides `Account` struct and Wallet/Backend interfaces. `Account` represents a Klaytn account located at a specific location defined by the optional URL field. `Wallet` represents a software or hardware wallet and `Backend` is a "wallet provider" that may  contain a batch of accounts they can sign transactions with
```
Comment (KimKyungup) [inline accounts/doc.go]: Just curious, what is difference between `blah` and "blah"?

Comment (andybclee) [reply accounts/doc.go]: @KimKyungup I think the double quotation marks of "wallet provider" should be removed. I used `Something` for structs and interfaces. " " was just for attention.

File: accounts/doc.go [hunk 3]
```diff
@@ -18,5 +18,19 @@
...
+/*
+Package accounts implements high level Klaytn account management.
+
+There are several important data structures and their hierarchy is as below
+ Manager -> Backend -> Wallet -> Account
+
+Source Files
+
+Each file provides the following features
+ - accounts.go	: Provides `Account` struct and Wallet/Backend interfaces. `Account` represents a Klaytn account located at a specific location defined by the optional URL field. `Wallet` represents a software or hardware wallet and `Backend` is a "wallet provider" that may  contain a batch of accounts they can sign transactions with
```
Comment (andybclee) [inline accounts/doc.go]:
```suggestion
 - accounts.go	: Provides `Account` struct and Wallet/Backend interfaces. `Account` represents a Klaytn account located at a specific location defined by the optional URL field. `Wallet` represents a software or hardware wallet and `Backend` is a wallet provider that may  contain a batch of accounts they can sign transactions with
```

File: accounts/doc.go [hunk 4]
```diff
@@ -18,5 +18,19 @@
...
+/*
+Package accounts implements high level Klaytn account management.
+
+There are several important data structures and their hierarchy is as below
+ Manager -> Backend -> Wallet -> Account
+
+Source Files
+
+Each file provides the following features
+ - accounts.go	: Provides `Account` struct and Wallet/Backend interfaces. `Account` represents a Klaytn account located at a specific location defined by the optional URL field. `Wallet` represents a software or hardware wallet and `Backend` is a wallet provider that may  contain a batch of accounts they can sign transactions with
```
Comment (jeonghonah) [inline accounts/doc.go]:
```suggestion
 - accounts.go	: Provides `Account` struct and Wallet/Backend interfaces. `Account` represents a Klaytn account located at a specific location defined by the optional URL field. `Wallet` represents a software or hardware wallet and `Backend` is a wallet provider that may contain a batch of accounts they can sign transactions with
```

Comment (andybclee) [reply accounts/doc.go]: @jeonghonah Wow.. an eagle's eye..

**Pull request merged**